e2e/constants.ts

```diff
@@ -15,3 +15,4 @@ export const VALID_PHONE_NUMBER = '+1' + faker.string.numeric(10);
 export const USER_NAME = 'test';
 export const MESSAGE = faker.lorem.sentence();
 export const BRAND_NAME = process.env.PLAYWRIGHT_BRAND_NAME;
+export const RANDOM_STRING = faker.string.alpha(10);
```

Comment:
This isn't a constant. It would change on every run. You should use `faker` directly, or if you have a complex fake, wrap it in a function you can invoke.

Reply:
done